jobs/ml_jobs/item_linkage/build_semantic_space.py

```diff
@@ -77,12 +79,23 @@ def make_batches(df: pd.DataFrame, batch_size: int):
             yield df[i : i + batch_size]
 
     db = lancedb.connect(MODEL_PATH)
-    db.drop_database()
-    db.create_table(
-        "items",
-        make_batches(df=items_df, batch_size=LANCEDB_BATCH_SIZE),
-        schema=ItemModel,
-    )
+    try:
+        logger.info("Creating LanceDB table...")
+        db.create_table(
+            "items",
+            make_batches(df=items_df, batch_size=LANCEDB_BATCH_SIZE),
+            schema=ItemModel,
+        )
```

Comment:
import lancedb

# Connect to your LanceDB database
db = lancedb.connect('path/to/your/database')

# Get the list of tables in the database
tables = db.list_tables()

# Check if 'items' table exists
if 'items' in tables:
    print("Table 'items' exists.")
else:
    print("Table 'items' does not exist.")